src/test/java/fi/helsinki/cs/tmc/core/commands/RunTestsTest.java

```diff
@@ -6,6 +6,9 @@
 import fi.helsinki.cs.tmc.core.exceptions.TmcCoreException;
 import fi.helsinki.cs.tmc.langs.domain.RunResult;
 
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
```

Comment:
Import order should be:
- All static imports in a single group
- fi.helsinki.cs.tmc imports (only if this source file is in the fi.helsinki.cs.tmc package space)
- Third-party imports, one group per top-level package, in ASCII sort order
- for example: android, com, junit, org, sun
- java imports
- javax imports